public/locales/en/translations.json

```diff
@@ -540,6 +540,50 @@
     "P7D": "Last week",
     "P1M": "Last month"
   },
+  "deployments": {
+    "title": "Deployments",
+    "flyouts": {
+      "new": {
+        "contextMenuName": "New deployment",
+        "title": "New deployment",
+        "apply": "Apply",
+        "cancel": "Cancel",
+        "close": "Close",
+        "type": "Package type",
+        "deviceGroup": "Device Group",
+        "name": "Name",
+        "priority": "Priority",
+        "package": "Package",
+        "typePlaceHolder": "Select package type",
+        "packagePlaceHolder": "Select package",
+        "deviceGroupPlaceHolder": "Select device group",
+        "priorityPlaceHolder": "Enter priority",
+        "namePlaceHolder": "Enter name",
+        "targetText": "targeted devices",
+        "infoText": "* Edge Manifest packages that you deploy will be applied to every edge device group in the device group and to any edge group that you add to the device group future. The deployment will run continously, verifying that current and future devices are configured correctly (and automatically updating that aren't). Package wont be applied to non-edge devices.",
```

Comment:
What is the asterisk for? Should it be an svg icon instead?

Reply:
No, it's red colored text asterisk as per design. Right now I'm just focusing on completing the flow. I will revisit this styling once the main flow is done.